uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -618,6 +618,29 @@ public void connect(String clientDeviceId, String componentId, String brokerId,
         throw lastException;
     }
 
+    /**
+     * Creates MQTT failed connection.
```

Comment:
Write right description like "Try to create MQTT connection to broker and ensure connection has been failed"